src/dev/flang/fe/SourceModule.java

```diff
@@ -969,13 +973,16 @@ public AbstractFeature lookupFeature(AbstractFeature outer, FeatureName name, Ab
    * outer's outer (i.e., use is unqualified), false to search in outer only
    * (i.e., use is qualified with outer).
    *
+   * @param sf the source file where type is used
```

Comment:
`where feature is used`, not `type`.

When can this be `null`, what does that mean?

Reply:
I removed this parameter, filtering the result afterwards instead.